widget/entry.go

```diff
@@ -409,6 +432,36 @@ func (e *Entry) TappedSecondary(pe *fyne.PointEvent) {
 	e.popUp.ShowAtPosition(popUpPos)
 }
 
+// findButton returns the button from a form with required style.
+// Used to find the OK or Cancel buttons on a form.
+func (e *Entry) findButton(requiredStyle ButtonStyle) *Button {
+	var btn *Button
+	driver.WalkVisibleObjectTree(
+		fyne.CurrentApp().Driver().CanvasForObject(e).Content(),
+		func(obj fyne.CanvasObject, _ fyne.Position, _ fyne.Position, _ fyne.Size) bool {
+			if w, ok := obj.(fyne.Disableable); ok && w.Disabled() {
+				// disabled widget cannot receive focus
+				return false
+			}
+			if b, ok := obj.(*Button); ok && b.Style == requiredStyle {
+				btn = b
+				return true
+			}
+			return false
+		}, nil)
+
+	return btn
+}
+
+func (e *Entry) HandleOkCancel() {
```

Comment:
I don't know what this does - looks like it taps 2 found buttons?
Pretty certain it should not be a public Entry API

Reply:
HandleOkCancel() is a orphaned function leftover from earlier tests. I have removed it.